website/addons/dryad/settings/defaults.py

```diff
@@ -0,0 +1,23 @@
+# DRYAD OAI-PMH metadata harvesting settings
+# Note where needed, format as (date, prefix, dataset)
+# At the moment, this part of the API is unused as the DataOne connection
+# provides more information
+DRYAD_OAI_IDENTIFY = 'http://www.datadryad.org/oai/request?verb=Identify'
+DRYAD_OAI_LISTSET = 'http://www.datadryad.org/oai/request?verb=ListSets'
+DRYAD_OAI_LISTMETADATAFORMAT = 'http://www.datadryad.org/oai/request?verb=ListMetadataFormats'
+DRYAD_OAI_LISTIDENTIFIERS = 'http://www.datadryad.org/oai/request?verb=ListIdentifiers&from={}&metadataPrefix={}&set={}'
+DRYAD_OAI_LISTRECORDS = 'http://www.datadryad.org/oai/request?verb=ListRecords&from={}&metadataPrefix={}&set={}'
+DRYAD_OAI_GETRECORD = 'www.datadryad.org/oai/request?verb=GetRecord&identifier={}&metadataPrefix={}'
```

Comment:
Also, consider building these urls from base urls, e.g.
```python
DRYAD_BASE_URL = 'http://www.datadryad.org/'

DRYAD_OAI_BASE = '{}oai/'.format(DRYAD_BASE_URL)
DRYAD_API_BASE = '{}mn/'.format(DRYAD_BASE_URL)
# etc.
```

Could also use `furl`, but that's not strictly necessary